src/views/Contributor.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <div>
+    <div class="container">
+      <div class="row markdown-body">
+        <div class="col-sm-8">
+          <h2>People</h2>
+          <h3>Committers</h3>
+          <SingleContributor v-for="(con,index) in conList" :key="'1' + index" :avatar="con.avatar" :name="con.name"
+                             :org="con.org" :role="con.role" :id="con.id" :date="con.date"></SingleContributor>
+          <h3>Contributors who submit PRs</h3>
```

Comment:
I think without this part in this page is fine, as our community continues to grow, we do n’t seem to have the energy to merge updates from many new contributors a day, and we ca n’t ask every contributor to be willing to update this page, especially users who just report issues.  What do you think? :)